casper/src/test/scala/coop/rchain/casper/util/ProtoUtilTest.scala

```diff
@@ -0,0 +1,57 @@
+package coop.rchain.casper.util
+
+import com.google.protobuf.ByteString
+import coop.rchain.casper._
+import coop.rchain.casper.protocol._
+import org.scalacheck.{Arbitrary, Gen}
+import org.scalacheck.Arbitrary.arbitrary
+import org.scalacheck.Gen.listOfN
+import org.scalatest._
+import org.scalatest.prop.GeneratorDrivenPropertyChecks
+
+class ProtoUtilTest extends FlatSpec with Matchers with GeneratorDrivenPropertyChecks {
+
+  val blockHashGen: Gen[BlockHash] = for {
```

Comment:
I want to merge those generators with `BlockGen` from `blockstore` and move to `models` in a follow-up PR